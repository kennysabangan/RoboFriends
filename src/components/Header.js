import React from 'react';
import CounterButton from './CounterButton';

const Header = () => {
  return (
    <>
      <h1 className='f1'>RoboFriends</h1>
      <CounterButton />
    </>
  )
}

export default Header;