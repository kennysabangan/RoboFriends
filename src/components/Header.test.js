import { shallow } from 'enzyme';
import React from 'react';
import Header from './Header';

it('renders Header without crashing', () => {
  expect(shallow(<Header />)).toMatchSnapshot()
})

