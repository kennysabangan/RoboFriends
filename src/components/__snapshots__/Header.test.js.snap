// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header without crashing 1`] = `
<Fragment>
  <h1
    className="f1"
  >
    RoboFriends
  </h1>
  <CounterButton />
</Fragment>
`;
